### DISCUSSION TITLE
Congratulate the performer

### DISCUSSION THREAD

**Discussion by nelsonmestevao** [cesium/karaokium#6] [Polls]
We have a voting systems that goes from 1 to 10 and that will probably continue to be the way to choose the winner. Nonetheless, we can work on some new festive ways to congratulate each performance.

**Comment by VitorLelis** [comment]
We could not only congratulate each performance individually but also congratulate the one with more specific reactions at the end of the event
For example, the performance that got more "hearts" reactions during the night will get the prize "Most loved"

**Reply by laraujo7** [reply]
Yeap, I think this idea is really fun 👌
